src/website/views/components/tabs.pug

```diff
@@ -126,28 +116,29 @@ h3 Icons
     ul.a-tabs.-vertical.-icons
       li.-active
         a(href='#')
-            <div class="a-icon -sm">
-              <svg>
-                <use xlink:href="#icon-atom"></use>
-              </svg>
-            </div>
-            <span>Active Tab</span>
+          .a-icon.-sm
```

Comment:
-sm class is not needed here as the icon is rendered sm by default. there's also two more instances of it below that need to be removed